### ISSUE TITLE
Rule idea: Adaptation of "function-calc-no-unspaced-operator" that supports function calls

### ISSUE THREAD

**Issue by mijelin** [kristerkari/stylelint-scss#270]
The [`function-calc-no-unspaced-operator`](https://stylelint.io/user-guide/rules/function-calc-no-unspaced-operator/) rule (from `stylelint`) is meant to do the following:

> Disallow an unspaced operator within `calc` functions.

In my codebase, the naming convention for functions is to use kebab-case -- however, the `-` characters in the function name are interpreted as minus signs, causing an error to be reported:

```scss
.some-element {
  // throws function-calc-no-unspaced-operator
  // because there is no space between "calculate" and "rems"
  width: calc(25% - calculate-rems(10));
}
```

It would be really awesome to have an SCSS-specific version of this rule that correctly handles this, so that we don't have to disable the rule for every similar use case.
